app/src/main/java/org/coepi/android/ui/symptoms/SymptomsViewModel.kt

```diff
@@ -60,8 +73,35 @@ class SymptomsViewModel(private val symptomRepo: SymptomRepo) : ViewModel() {
     fun onChecked(symptom: SymptomViewData) {
         checkedSymptomTrigger.onNext(symptom)
     }
+    fun doSubmitSymptms(){
+        log.i("About to send sypmtom:");
+        if( realmprovider!= null ){
```

Comment:
You have to remove `realmProvider` from here, but in general: In Kotlin nullable values have a `?` at the end. If it doesn't it means that it can't be `null` and you don't need these checks.

Reply:
injectin repo removed also these lines